packages/salesforcedx-utils-vscode/src/commands/notificationService.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Observable } from 'rxjs/Observable';
+import { workspace } from 'vscode';
```

Comment:
lets merge these and have only one import for `vscode`

Reply:
Updated - see https://github.com/forcedotcom/salesforcedx-vscode/pull/2660/commits/1b4a811e1eaf5e8e52575b8d550724bd598664ed